### PR TITLE
WSL instructions

### PR DESCRIPTION
**Pull request by eteq**
This re-jiggers the instructions to recommend using WSL on windows instead of the native windows application (and simplifies the instruction by removing some of the "if windows do X if on linux/mac do Y" clauses)

### CODE REVIEW

**Comment by eteq** [comment]
The CI failure has nothing to do with this PR, as it doesn't touch any of the notebooks.

File: 00-Install_and_Setup/README.md [hunk 1]
```diff
@@ -95,19 +103,11 @@ The python version and all needed packages, including astropy, are listed in the
 
 On Mac or Linux, open your terminal.
```
Comment (wtgee) [inline 00-Install_and_Setup/README.md:96]: Still have an OS specific reference here.

File: 00-Install_and_Setup/README.md [hunk 2]
```diff
@@ -95,19 +103,11 @@ The python version and all needed packages, including astropy, are listed in the
...
 following.
 
-On a Mac/Linux:
-
-    % cd astropy-workshop/00-Install_and_Setup/
-
-On Windows:
-
-    % cd astropy-workshop\00-Install_and_Setup\
+% cd astropy-workshop/00-Install_and_Setup/
```
Comment (wtgee) [inline 00-Install_and_Setup/README.md:110]: Needs code wrapping.

**Review by wtgee — APPROVED** [review]
Approving but I think a few minor changes. There's also one more reference to opening the Anaconda Prompt in the file that didn't make this edit, which does not seem intentional but may be.

File: 00-Install_and_Setup/README.md (hunk 1 above)
Comment (stargaser) [reply 00-Install_and_Setup/README.md:96]: Better to say, "Open a terminal"? Although I guess on Windows that is within WSL?

**Pull request merged**